components/browser/engine-gecko/src/test/java/mozilla/components/browser/engine/gecko/GeckoResultTest.kt

```diff
@@ -16,9 +16,12 @@ import org.junit.Assert.fail
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.mozilla.geckoview.GeckoResult
+import org.robolectric.annotation.LooperMode
 
+@Suppress("DEPRECATION")
 @ExperimentalCoroutinesApi
 @RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.LEGACY)
```

Comment:
Could we clarify what exactly we are suppressing the deprecation on ? I will be good to just add the deprecation on the specific item and link a ticket to letter address the issue :)

Reply:
I added a comment to clarify the deprecation suppression is for the LEGACY LooperMode.